src/modules/dashboard/components/ActionsPage/StakingWidget/StakingSlider.tsx

```diff
@@ -161,7 +161,7 @@ const StakingSlider = ({
           name="amount"
           value={values.amount}
           limit={userStakeLimitPercentage}
-          step={0.01}
+          step={1}
```

Comment:
I agree that this may take care of those edge cases where you try to stake 100% and you stake 99% instead. However, I think this comes with the tradeoff that it offers less flexibility or specificity when staking.

I'm not sure whether it's worth it in comparison with this

Reply:
Addressed through Discord